prometheus-server-metrics/src/main/scala/org/http4s/server/prometheus/PrometheusMetrics.scala

```diff
@@ -102,18 +103,24 @@ object PrometheusMetrics {
       _ <- emptyResponseHandler.traverse_(status =>
         Sync[F].delay {
```

Comment:
I very minor thing, would it help to use an implicit parameter here like

```scala
private def onEmpty[F[_]](
  m: Method,	       m: Method,
  start: Long,	       start: Long,
  headerTime: Long,	       headerTime: Long,
  serviceMetrics: ServiceMetrics,	       serviceMetrics: ServiceMetrics,
  emptyResponseHandler: Option[Status]	       emptyResponseHandler: Option[Status]
)(implicit F: Sync[F]): F[Unit] =
```

it could save some implicit search in positions like

```scala
now <- Sync[F].delay(System.nanoTime)
```
?
apologies if it has been already discussed

Reply:
so I guess here the preference is not to use something like 
```scala
now <- F.delay(System.nanoTime)
```
but is to use the implicit search of the `apply` on `Sync[F]`. 
Thanks 👍

Reply:
Well it's my preference as I like to clarify which constraints I'm using. Which allows me to more easily see when I can upon removing a constrain use downgrade. Its just my personal practice.

Reply:
makes sense to me. Thanks 👍